.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+Thanks for contributing to RIOT!
+
+
+### Preliminary checks
+
+The RIOT community cares a lot about code quality.
```

Comment:
this whole section is way to *chatty* for my taste. Why not have checkboxes with short descriptions, and thats it.

Like:

### Preliminary checks

- [ ] code complies to [RIOT coding conventions](https://github.com/RIOT-OS/RIOT/wiki/Coding-conventions)
- [ ] no errors reported by `make static-test` (run from repository root)
- [ ] PR is **not** *work-in-progress* (WIP) nor relies on any other PR

Further information can be found in the RIOT [contributing guide](../CONTRIBUTING.md).

Reply:
A welcome message is important for newcomers and dheckboxes are useless when you write the PR description. That's why I removed them.

I also think that it's important to write things correctly (or 'chatty' if you prefer) for people who are not used to open-source contributions as you are.